spec/javascripts/NewService/formElements/__snapshots__/ServiceManualListItems.spec.tsx.snap

```diff
@@ -106,11 +116,22 @@ exports[`should render required form fields with errors 1`] = `
       type="text"
       value="new api"
     />
-    <p
-      className="inline-errors"
-    >
-      Only ASCII letters, numbers, dashes and underscores are allowed.
-    </p>
+    <HelperText>
+      <HelperTextItem>
+        <p
+          className="inline-errors"
+        >
+           
+          Only ASCII letters, numbers, dashes and underscores are allowed.
```

Comment:
I suggest to change the error message for "Invalid", since it's a bit confusing to see the same hint twice in the snapshot.

Reply:
![image](https://user-images.githubusercontent.com/84672731/233915486-b69cde88-d2b7-4edc-8771-4c1ea0898b2a.png)
Looks better now

Reply:
but  still see in snapshot,

Reply:
only Invalid?